rigex-front-end/src/components/RigSidebar/RigList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { List, ListItemText, MenuItem } from '@material-ui/core';
+
+import './sideBar.scss';
+
+function RigList(props) {
+  const [selectedIndex, setSelectedIndex] = React.useState(0);
+  const handleListItemClick = (event, index) => {
+    setSelectedIndex(index);
+  };
+
+  props.rigList.sort((a, b) => (a.name > b.name ? 1 : -1));
```

Comment:
This will only sort on initial component render. If there is behavior that would necessitate a resort throughout the lifetime of the component, it won't happen because of the current scope

Reply:
https://reactjs.org/docs/hooks-reference.html#useeffect